engine/examples/dotnetcore.md

```diff
@@ -0,0 +1,102 @@
+---
+description: Create a Docker image by layering your ASP.NET Core app on debian
+for Linux Containers or with Windows Nano Server containers using a Dockerfile.
+keywords: docker, dockerize, dockerizing, dotnet, .NET, Core, article, example,
+platform, installation, containers, images, image, dockerfile, build, asp.net,
+asp.net core title: Dockerizing a .NET application
+---
+
+## Introduction
+
+This example demonstrates how to dockerize an ASP.NET Core application.
+
+## Why build ASP.NET Core?
+- [Open-source](https://github.com/aspnet/home)
+- Develop and run your ASP.NET Core apps cross-platform on Windows, MacOS and
+  Linux
+- Great for modern cloud-based apps, such as web apps, IoT apps and mobile
+  backends 
+- ASP.NET Core apps can run on [.NET
+  Core](https://www.microsoft.com/net/core/platform) or on the full [.NET
+  Framework](https://www.microsoft.com/net/framework)
+- Designed to provide an optimized development framework for apps that are
+  deployed to the cloud or run on-premise
+- Modular components with minimal overhead retain flexibility while 
+  constructing
+  your solutions
+
+## Prerequisites
+This example assumes you already have an ASP.NET Core app 
+on your machine. If you are new to ASP.NET you can follow a [simple 
+tutorial](https://www.asp.net/get-started) to initialize a project or clone our [ASP.NET Docker Sample](https://github.com/dotnet/dotnet-docker-samples/tree/master/aspnetapp).
+
+## Create a Dockerfile for an ASP.NET Core application
+1.  Create a `Dockerfile` in your project folder.
+2.  Add the text below to your `Dockerfile` for either Linux or [Windows
+   Containers](https://docs.microsoft.com/en-us/virtualization/windowscontainers/about/).
```

Comment:
no problem, it was just a nit anyway; perhaps you're right and it's due to it being a link that's wrapped. good call!